src/data/reducers/loginForm.js

```diff
@@ -0,0 +1,35 @@
+import {
+  LOGIN_FAILURE,
+  LOGIN_REQUEST,
+  LOGIN_SUCCESS,
+} from '../constants/loginForm';
+
+const loginForm = (state = {
+  isAuthenticated: localStorage.getItem('access_token') !== false,
+  isLoginSuccess: false,
+  loading: false,
+  error: '',
+}, action) => {
+  switch (action.type) {
+    case LOGIN_REQUEST:
+      return {
+        ...state,
+        loading: action.payload.loading,
```

Comment:
You can just set `loading: true,` here instead of passing in a boolean through the action payload (see the `courseEnrollments.js` reducer).

Reply:
If loading is set to true then how will the spinner stop? I believe it should stay like this unless I'm missing something

Reply:
@jonalexvasquez On `LOGIN_REQUEST`, loading will always be true. Then, in the reducers for `LOGIN_SUCCESS` and `LOGIN_FAILURE` actions, you would set `loading: false`.